iceoryx_hoofs/posix/time/source/deadline_timer.cpp

```diff
@@ -48,11 +51,21 @@ iox::units::Duration deadline_timer::remainingTime() const noexcept
     {
         return m_endTime - currentTime;
     }
-    return iox::units::Duration(std::chrono::milliseconds(0));
+    using namespace iox::units::duration_literals;
+    return 0_s;
 }
 
 iox::units::Duration deadline_timer::getCurrentMonotonicTime() noexcept
 {
-    return iox::units::Duration{std::chrono::steady_clock::now().time_since_epoch()};
+    timespec time_since_epoch{0, 0};
+    if (posix::posixCall(clock_gettime)(CLOCK_REALTIME, &time_since_epoch)
```

Comment:
Not sure but doesn't `std::chrono::steady_clock` correspond more to `CLOCK_MONOTONIC`?

Reply:
Yes, it must be `CLOCK_MONOTONIC`